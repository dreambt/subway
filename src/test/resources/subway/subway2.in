L2-8,L5-7