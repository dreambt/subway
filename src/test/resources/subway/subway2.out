2:L2-8,X2,X3,L5-7